<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>protocolDialog</class>
 <widget class="QDialog" name="protocolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/images.qrc">
    <normaloff>:/general/general/协议.png</normaloff>:/general/general/协议.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>698</width>
        <height>489</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="lbiregList">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>寄存器表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabRegList">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>285</height>
          </size>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>15</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>14</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>13</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>12</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>11</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>10</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>9</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>8</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>7</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>6</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>5</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>0</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0x03E8</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>设备标志寄存器</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>0x03E9</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>通信参数1</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>0x03EA</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>通信参数2</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>0x03EB</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>通信参数3</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>0x03EC</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>通信参数4</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>0x03ED</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>系统寄存器</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>0x03EE</string>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>输出通道A</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>0x03EF</string>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string>输出通道B</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>0x03F0</string>
          </property>
         </item>
         <item row="8" column="1">
          <property name="text">
           <string>校准通道A</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>0x03F1</string>
          </property>
         </item>
         <item row="9" column="1">
          <property name="text">
           <string>校准通道B</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
